lib/utils/__tests__/isStandardSyntaxDeclaration.test.js

```diff
@@ -1,241 +1,179 @@
 'use strict';
 
 const isStandardSyntaxDeclaration = require('../isStandardSyntaxDeclaration');
-const less = require('postcss-less');
 const postcss = require('postcss');
-const sass = require('postcss-sass');
-const scss = require('postcss-scss');
+const postcssLess = require('postcss-less');
+const postcssSass = require('postcss-sass');
+const postcssScss = require('postcss-scss');
 
 describe('isStandardSyntaxDeclaration', () => {
 	it('standard prop and value', () => {
-		decls('a { a: b }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeTruthy();
-		});
+		expect(isStandardSyntaxDeclaration(decl('a { a: b }'))).toBeTruthy();
```

Comment:
Could we use `toBe(true)` and `toBe(false)` for a more strict comparison, please?